.github/workflows/tests.yml

```
@@ -19,6 +19,7 @@ jobs:
           bazelisk build --local_test_jobs=1 -- //... -//tests/ios/frameworks/mixed-source/sources-with-prebuilt-binaries/...
           bazelisk test --local_test_jobs=1  //tests/macos/...
           bazelisk test --local_test_jobs=1 --apple_platform_type=ios //tests/ios/...
```

```suggestion
          bazelisk test --local_test_jobs=1 -- //... -//tests/ios/... -//tests/macos/...
          bazelisk test --local_test_jobs=1 --apple_platform_type=macos -- //tests/macos/...
          bazelisk test --local_test_jobs=1 --apple_platform_type=ios -- //tests/ios/...
```

This way tests outside of the macos/ios dirs get run (such as the tests for the hmap tool)